### PR TITLE
v3 of the module

### PR DESCRIPTION
**Pull request by shigmas**
To avoid confusion, since this is a rewrite, we will keep development on the v2 module simultaneously. This allows us to have a master branch were we can easily see both v2 and v3 modules.
Tweaks to run with v3

Most of the code was v3, so just a few more changes to get all the versions correct.

For reference, the previous PR:
https://github.com/moderepo/device-sdk-go/pull/27

### CODE REVIEW

**Review by takeshinoda — APPROVED** [review]
Thank you for your great working!!

File: v3/README.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# MODE Device SDK for Go
+
+[![GoDoc](https://godoc.org/github.com/moderepo/device-sdk-go?status.svg)](https://godoc.org/github.com/moderepo/device-sdk-go)
```
Comment (ethankan) [inline v3/README.md]: Need to update this link?

File: v3/README.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        }
+    }
+```
+
+See more examples [here](https://github.com/moderepo/device-sdk-go/blob/master/examples).
+
+
+## Documentation
+
+See the full API documentation [here](https://godoc.org/github.com/moderepo/device-sdk-go).
```
Comment (ethankan) [inline v3/README.md]: And this?

**Review by ethankan — CHANGES_REQUESTED** [review]
My only request is to update the README a little bit.

File: v3/README.md (hunk 2 above)
Comment (shigmas) [reply v3/README.md]: points to v3 now. Is that it @ethankan ?

**Review by ethankan — APPROVED** [review]

**Pull request merged**